app/bridge/server.js

```diff
@@ -46,14 +56,42 @@ function createServer(name, version, methods) {
       interface: this.getContract()
     });
 
+    // we keep a ref to the listener to be able to remove it.
+    channel.onMessageListener = e => {this.onmessage.call(this, channel, e.data);};
```

Comment:
Do we really need to remove it or can we just close the channel and it will be gone ?